src/api/procedures/inviteAccount.ts

```diff
@@ -81,7 +81,7 @@ export async function prepareInviteAccount(
 
   if (hasPendingAuth) {
     throw new PolymeshError({
-      code: ErrorCode.ValidationError,
+      code: ErrorCode.NoDataChange,
```

Comment:
`NoDataChange` refers only to data mutations in which the user is attempting to change a value to the same existing one. I went with `UnmetPrerequisite` for asset creation because a prerequisite to create an asset is for that asset to not already exist

Reply:
But it is very very similar to this case. A prerequisite to invite an account is for that account to not already be invited. As I see these cases, they are the same.